packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -888,20 +893,21 @@ class CrosshairsTool extends AnnotationTool {
       vec2.scale(
         canvasVectorFromCenterMid,
         canvasUnitVectorFromCenter,
-        canvasDiagonalLength * 0.25
+        canvasMinDimensionLength * 0.4
```

Comment:
I'm not sure why it was 0.25 and not you are suggesting 0.4, same for below

Reply:
Let's see if I can explain myself...

Notice that I'm not using the diagonal, but the shorter side of the canvas because currently it can happen that they are out of canvas. In my screenshot of the description, you can see that only the controls of the slab thickness are visible, the rotation controls are out. And, it could also happen that all are out depeding on the canvas dimensions ratio, forcing you to pan the image or change the crosshair center point to be able to use the controls.

To ensure that at least the controls of one side are visible, the outer ones (rotation) should be placed at half the length of the shorter side. **Thus, should be 0.5 at most, and I decided to use 0.4 to have some margin to the edge. 0.2 is because it's in the middle of 0 and 0.4.**

Reply:
This is great explanation, thanks!
Can you put this comment in the code as well then we can merge

Reply:
Great. Done!